app/assets/stylesheets/app/public_dashboard.scss

```diff
@@ -0,0 +1,54 @@
+$pd-card-width: 40%;
+$pd-card-margin: $pd-card-width / 8;
+
+.pd-card {
+  @extend .card;
+  width: $pd-card-width;
+  margin: $pd-card-margin 0 0 $pd-card-margin;
+  padding: 1% 0;
+  display: flex;
+  flex-direction: column;
+
+  &__container {
+    display: flex;
+    flex-wrap: wrap;
+    padding-bottom: $pd-card-margin;
+    height: 100vh;
+  }
+
+  &__upper-half {
+    flex: 1;
+  }
+
+  &__lower-half {
+    flex: 1;
```

Comment:
Properties should be ordered align-items, display, flex, justify-content, padding